### PR TITLE
Add changes from master to production

### PR DESCRIPTION
**Pull request by WilielFlorenciani**
The process requires scrubbing development variables before merging into prod

### CODE REVIEW

File: src/Schedule/UpdateEvent.js [hunk 1]
```diff
@@ -197,10 +182,10 @@ function ScheduleUpdateEvent (props) {
             {event && (
                 <div className="addNewEvent">
                     
-                        <div key={event.meetid}>  
+                        <div key={event.projectid}>
```
Comment (WilielFlorenciani) [inline src/Schedule/UpdateEvent.js]: please update to have this be meetid

File: src/config/config.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
-const baseURL = 'https://iapshowroom.ece.uprm.edu';
-const WebSocketURL = 'wss://iapshowroom.ece.uprm.edu/ws/'
+const baseURL = 'http://localhost:8080';
```
Comment (WilielFlorenciani) [inline src/config/config.js]: add prod variables once again

File: src/HomeArea/Home.js [hunk 3]
```diff
@@ -6,7 +6,7 @@ import { useHistory } from "react-router-dom";
 import { useState} from 'react';
 import axios from "../context/axios";
 const config = require('../config/config');
```
Comment (WilielFlorenciani) [inline src/HomeArea/Home.js]: change to import

**Review by WilielFlorenciani — COMMENTED** [review]
Changes are ready for production

**Review by DanielTorres10 — APPROVED** [review]

**Review by WilielFlorenciani — COMMENTED** [review]
conflicts solved

**Pull request merged**